client/lib/recommended-sites-store/store.js

```diff
@@ -30,7 +30,7 @@ function receiveRecommendations( data ) {
 	// consume the recommendations
 	const previousRecs = recommendations;
 	if ( data && data.blogs ) {
-		if ( data.blogs.length === 0 ) {
+		if ( data.blogs.count === 0 ) {
```

Comment:
but the thing i don't understand is how data.blogs is becoming an immutable instance...